chainer/links/connection/convolution_nd.py

```diff
@@ -82,7 +133,11 @@ def __init__(self, ndim, in_channels, out_channels, ksize, stride=1, pad=0,
                 initial_bias = initializers._get_initializer(initial_bias)
                 self.b = variable.Parameter(initial_bias, out_channels)
 
-    def __call__(self, x):
+    def _initialize_params(self, in_channels):
+        W_shape = (self.out_channels, in_channels) + self.ksize
```

Comment:
Do you have to take `self.groups` into account, as it's done in `Convolution2D`?